gatling-core/src/main/scala/io/gatling/core/json/Json.scala

```diff
@@ -45,7 +49,7 @@ object Json {
   }
 
   private def writeString(s: String, rootLevel: Boolean) = {
-    val escapedLineFeeds = s.replace("\n", "\\n")
+    val escapedLineFeeds = objectMapper.writeValueAsString(s).replaceAll("^\"|\"$", "")
```

Comment:
> We should fix the way gatling build JSON probably.

We can't do that. It's intended to support re-injecting a captured JSON tree in a node position in a Gatling EL template.